npc/re/custom/lasagna/lasagna_mob.txt

```diff
@@ -0,0 +1,21 @@
+// Inaccurate
```

Comment:
Should be split up to dungeons/fields like all others.
Script header is missing.
Field seperators are missing too:
```
//==================================================
// lasa_fild01 - Lasagna Field
//==================================================
```